ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -53,16 +53,36 @@ const ResultGroup = styled.div`
   }
 `
 
+const DidYouMeanContainer = styled.div`
+  text-align: center;
+`
+
+const ShowMore = styled.a`
+  padding: 0.25rem;
+  font-size: 12px;
+`
+
+const ResendQuery = styled.a`
```

Comment:
It's a little strange to me to use anchor tags for this, but should be fine for now.